Dockerfile

```diff
@@ -18,16 +17,11 @@ RUN crudini --merge /etc/ironic/ironic.conf < /tmp/ironic.conf && \
 
 COPY ./runironic-api.sh /bin/runironic-api
 COPY ./runironic-conductor.sh /bin/runironic-conductor
-COPY ./runironic-exporter.sh /bin/runironic-exporter
 COPY ./rundnsmasq.sh /bin/rundnsmasq
 COPY ./runhttpd.sh /bin/runhttpd
 COPY ./runmariadb.sh /bin/runmariadb
 COPY ./configure-ironic.sh /bin/configure-ironic.sh
 
-# TODO(dtantsur): remove these 3 scripts and squash runexporterapp into
-# runironic-exporter if we decide to stop supporting running all 3 processes
-# via one entry point.
```

Comment:
Please update this TODO instead of removing it as it also applies to runhealthcheck and runironic

Reply:
done